data/dockerFiles/fedoraDockerFile/Dockerfile

```diff
@@ -0,0 +1,96 @@
+#  This file is part of cc-oci-runtime.
+#
+#  Copyright (C) 2016 Intel Corporation
+#
+#  This program is free software; you can redistribute it and/or
+#  modify it under the terms of the GNU General Public License
+#  as published by the Free Software Foundation; either version 2
+#  of the License, or (at your option) any later version.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with this program; if not, write to the Free Software
+#  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+#
+
+# XXX: named version (also referenced below for the OBS elements)
+FROM fedora:24
```

Comment:
I wonder if there is a way to create variables that can be used in `Dockerfile`s? If so, we can avoid hard-coding the version of Fedora throughout the file which makes it more error-prone to maintain.

If not, we should consider auto-generating `Dockerfile` from `Dockerfile.in` (or `Dockerfile.fedora.in`) and defining the version of Fedora/Ubuntu we support in https://github.com/01org/cc-oci-runtime/blob/master/versions.txt. We could add something like:

```
docker_fedora_version=24
docker_ubuntu_version=16.04
```
... and then update `Makefile.am` and `configure.ac` accordingly.